### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by hadro**

### CODE REVIEW

File: iiify/templates/docs/overview.html [hunk 1]
```diff
@@ -1,242 +1,94 @@
...
-    server: "ia800309.us.archive.org",
-    uniq: 893310999,
-    updated: 1442380951,
-    workable_servers: []
-  }
-    </code></pre>
+    <p>The Image API creates images using the following template: </p>
+      <div class="request">{scheme}://{server}/{prefix}/{identifier}/{region}/{size}/{rotation}/{quality}.{format}</div>
+    <p>In the context of the Internet Archive items, that will look like this for a full image:</p>
+    <div class="request">https://iiif.archive.org/image/iiif/3/<b>:id+filename</b>/full/max/0/default.jpg</div>
```
Comment (glenrobson) [inline iiify/templates/docs/overview.html:67]: Could we add a comment to say the id+filename combination can be got from the download link? For example if the file download link is:

https://archive.org/download/img-8664_202009/IMG_8664.jpg

then the IIIF image API URL would be (with / encoded as %2f):

https://iiif.archive.org/image/iiif/3/img-8664_202009%2fIMG_8664.jpg

I know it adds complexcity to the description but its an implementation detail that can easily be forgotten and if people are looking at how to genereate the Image API then knowing how it works will be useful.

**Comment by glenrobson** [comment]
Link to documentation: https://internetarchive-iiif-update-docs.ux-fnf-misc.archive.org/iiif/documentation

File: iiify/templates/docs/overview.html (hunk 1 above)
Comment (hadro) [reply iiify/templates/docs/overview.html:67]: Ok, I've put in a few lines about this, thanks Glen

**Review by glenrobson — APPROVED** [review]
Thanks looks good to me.

**Pull request merged**